### ISSUE TITLE
Default curl values

### ISSUE THREAD

**Issue by ddieruf** [SteeltoeOSS/Initializr#78]
I ran this:
```bash
curl https://start.steeltoe.io/starter.zip -o PiggyMetrics.zip -d projectName=piggymetrics -d dependencies=actuators,discovery,dynamiclogger,configserver,mongodb,oauthconnector
```
And it gave me this VS project type:
```xml
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>netcoreapp2.2</TargetFramework>
    <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
    </PropertyGroup>

  <ItemGroup >
    <PackageReference Include="Microsoft.AspNetCore.App"/>
    <PackageReference Include="Steeltoe.Extensions.Configuration.ConfigServerCore" Version="2.3.0" />
    <PackageReference Include="Steeltoe.Management.ExporterCore"  Version="2.3.0"/>
    <PackageReference Include="Steeltoe.Management.CloudFoundryCore" Version="2.3.0" />
    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
    <PackageReference Include="Steeltoe.Discovery.ClientCore" Version="2.3.0"/>
    <PackageReference Include="Steeltoe.CloudFoundry.ConnectorCore"  Version="2.3.0"/>
    <PackageReference Include="MongoDB.Driver" Version="2.8.1" />
    <PackageReference Include="Steeltoe.Extensions.Logging.DynamicLogger" Version="2.3.0"/>
  </ItemGroup>
</Project>
```

It should have matched the defaults of the UI. Steeltoe packages `2.4.x`  and `netcoreapp3.1`.

**Comment by ddieruf** [comment]
After updating all nuget to latest, the project looked like this:
```xml
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>netcoreapp3.1</TargetFramework>
    <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
    </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.App" />
    <PackageReference Include="Steeltoe.Extensions.Configuration.ConfigServerCore" Version="2.4.2" />
    <PackageReference Include="Steeltoe.Management.ExporterCore" Version="2.4.2" />
    <PackageReference Include="Steeltoe.Management.CloudFoundryCore" Version="2.4.2" />
    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
    <PackageReference Include="Steeltoe.Discovery.ClientCore" Version="2.4.2" />
    <PackageReference Include="Steeltoe.CloudFoundry.ConnectorCore" Version="2.4.2" />
    <PackageReference Include="MongoDB.Driver" Version="2.10.2" />
    <PackageReference Include="Steeltoe.Extensions.Logging.DynamicLogger" Version="2.4.2" />
  </ItemGroup>
</Project>
```

**Comment by ccheetham** [comment]
Fixed as part of work for #68
